### PR TITLE
[Inductor] Fix lowmem_dropout() missing 1 required positional argument: 'p'

### PR DESCRIPTION
**Pull request by yanboliang**
Fixes error from 7k github models: https://github.com/jansel/pytorch-jit-paritybench/blob/master/generated/test_GuYuc_WS_DAN_PyTorch.py

Error:
```
TypeError: lowmem_dropout() missing 1 required positional argument: 'p'
       
While executing %lowmem_dropout : [#users=1] = call_function[target=torch._inductor.overrides.lowmem_dropout](args = (%avg_pool2d_9,), kwargs = {training: False})
```

cc @mlazos @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

File: test/inductor/test_torchinductor.py [hunk 1]
```diff
@@ -3677,7 +3677,7 @@ def test_dropout(self):
 
         @torch._dynamo.optimize("inductor")
         def fn(a):
-            return torch.nn.functional.dropout(a, 0.5, True)
+            return torch.nn.functional.dropout(a)
```
Comment (ngimel) [inline test/inductor/test_torchinductor.py:3680]: I think we should test both invocations that specify p, and that don't.

**Review by ngimel — APPROVED** [review]

File: test/inductor/test_torchinductor.py (hunk 1 above)
Comment (yanboliang) [reply test/inductor/test_torchinductor.py:3680]: SG, will add the other test!

**Review by jansel — APPROVED** [review]

**Review by desertfire — APPROVED** [review]

**Pull request closed without merging**